timeseries/src/autogluon/timeseries/models/local/statsforecast.py

```diff
@@ -21,14 +22,35 @@ def _update_local_model_args(self, local_model_args: Dict[str, Any]) -> Dict[str
     def _get_model_type(self) -> Type:
         raise NotImplementedError
 
+    def _get_local_model(self, local_model_args: Dict):
+        model_type = self._get_model_type()
+        return model_type(**local_model_args)
+
+    def _get_point_forecast(
+        self,
+        time_series: pd.Series,
+        local_model_args: Dict,
+    ):
+        return self._get_local_model(local_model_args).forecast(
+            h=self.prediction_length, y=time_series.values.ravel()
+        )["mean"]
+
     def _predict_with_local_model(
```

Comment:
Why not make this method abstract (raise `NotImplementedError`) for the abstract class, and then implement it for each of `AbstractProbabilisticStatsForecastModel` and `AbstractConformalizedStatsForecastModel`?

Reply:
I agree.